lib/clipboard.coffee

```diff
@@ -1,6 +1,6 @@
 Clipboard =
   _createTextArea: ->
-    textArea = document.createElement("textarea")
+    textArea = DomUtils.createElement("textarea")
```

Comment:
Moving all of the `createElement` calls to DomUtils seems like a good idea.

However, perhaps not this one (which is only used from the background page).  It seems to break yank from visual mode:

- `v`, select some text, the `y` seems not to be working.